tests/testCampaign.py

```diff
@@ -0,0 +1,112 @@
+from deltamail.campaign import BulkMailCampaign, TransactionMailCampaign
```

Comment:
That part Tinu is handling. He's working on the code right now. But yes, I'll add a todo.